### ISSUE TITLE
What analyzers go to the PostgreSQL database for feeds?

### ISSUE THREAD

**Issue by almaz045** [dependency-check/DependencyCheck#6560]
Hello! I supplement the feeds in the postgresql database after initialization with the init.sql script from the instructions. This is for personal purposes. My question is, what analyzers will go into this postgresql database to take feeds from there (accordingly, take my views too)?
I start DC with the following command:
dependency-check.sh -o . -s. -f HTML --dbDriverName org.postgresql.Driver --connectionString 'jdbc:postgresql://127.0.0.1:5432/depcheck?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC' --dbPassword postgres --dbUser postgres --dbDriverPath ./postgresql-42.5.4.jar --disableOssIndex --disableCentral

The second question is, do I fill the database correctly so that DC can find my feeds?
To add my own feeds I used the following commands:
insert into **vulnerability** (cve, description, v2severity, v2score, v3basescore, v3baseseverity) values ('CVE-2022-9998', E'SOME-description', 'HIGH', '8.0', '8.0', 'HIGH');
insert into **cpeentry** (part, vendor, product, version, update_version, edition, lang, sw_edition, target_sw, target_hw, other, ecosystem) values ('a', 'sweetalert2_project','sweetalert2','*', '*', '*','*','*','node.js','*','*','nodejs');
insert into **software** (cveid, cpeentryid, versionstartincluding, vulnerable) values ((select id from vulnerability where cve='CVE-2022-9998'), (select id from cpeentry where product='sweetalert2' limit 1),'11.4.9 ', true);

**Comment by chadlwilson** [comment]
I don't believe this is a design goal of ODC to have you directly inserting things into the DB.
